biorbd_optim/problem_type.py

```diff
@@ -208,14 +208,64 @@ def get_data_from_V(ocp, V, num_phase=None):
         else:
             return q, q_dot, tau
 
+    @staticmethod
+    def get_states_integrated_from_V(ocp, V, number_elements=1, concatenate_phases=False):
+        v = ProblemType.get_data_from_V(ocp, V)[:3]
+
+        if ocp.nb_phases == 1:
+            v = [v]
```

Comment:
It feels that problem_type is a sketchy place for this method... Is OCP not better?